kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/ModemManagerDbusWrapper.java

```diff
@@ -40,17 +40,19 @@ public class ModemManagerDbusWrapper {
 
     private static final Logger logger = LoggerFactory.getLogger(ModemManagerDbusWrapper.class);
 
-    private static final String MM_BUS_NAME = "org.freedesktop.ModemManager1";
+    private static final String NETWORK_MANAGER_BUS_NAME = "org.freedesktop.NetworkManager";
+    private static final String MM_DEVICE_NAME = "org.freedesktop.NetworkManager.Device";
+    private static final String MODEM_MANAGER_BUS_NAME = "org.freedesktop.ModemManager1";
```

Comment:
Following the convention already used in this class, the name should be `MM_BUS_NAME`